pages/inspector/inspector-issues-view.mdx

```diff
@@ -25,22 +26,91 @@ Each issue in the table represents an issue on a particular event, or a particul
+  to filter by, please [let us know](/help/troubleshooting).
+</Callout>
+
+## Issue details
+
+Click any issue in the list to see more details. If you have multiple sources connected with Inspector, you can dig into which sources are impacted by the issue and uncover which sources are most severely affected.
+
+<br />
+<center>
+  <img
+    src="/docs/images/inspector/inspector-issue-details.png"
+    width="500px"
+    alt="Image shows example issue details for an unexpected property."
+  />
+</center>
+
+### Event info
+
+The event impacted by the issue, its volume and whether it is documented in your Avo tracking plan.
+
+- If the event is present in the tracking plan, you can navigate to it to review and/or update the event definition
+- If the event is missing from your tracking plan, you can add it to your tracking plan.
+
+### Property info
+
+The property impacted by the issue, and whether it is documented to be sent with the event in your tracking plan.
+
+- If the property is present on the event in your tracking plan, you can navigate to it to review and/or update.
+- If the property is present in your tracking plan but not set to be sent with your event, you can add it to the event definition.
+- If the property is missing from your tracking plan, you can add it to your tracking plan.
+
+<Callout type="info">
+  Not all issues are related to properties. See [full list of issue types
+  here](/explore-tracking-plan/issue-types-in-inspector).
+</Callout>
+
+### Releases
+
+The releases impacted by the issue in the last 24 hours, sorted by the most recent release first. For each release you will see:
+- The release version (hover over the clock icon to see when that particular release was first seen (release date))
+- The event volume seen on that release in the past 24 hours
+- The issue volume seen on that release in the past 24 hours
+- The issue percentage seen on that release in the past 24 hours
+
+### Issue across sources
+
+At the bottom of your issue details, you will see a list of all sources that are impacted by the issue. For each source you will see:
+Event volume, issue volume and issue percentage and a link to the issue details on the other sources.
+
+![An example iOS issue that also impacts Android](/images/inspector/inspector-issue-details-sources.png)
+
+## Release and source breakdown
+
+By clicking the "X releases on Y sources" button right above the issues table, you can see a breakdown of the releases we saw in the past 24 hours and we used to calculate the issues you're looking at.
+
+!["X releases on Y sources" button"](/images/inspector/inspector-issues-release-breakdown.png)
+
+This breakdown can be helpful to understand which releases have been fully rolled out and you can rely on the data from and which releases are still being rolled out and you could expect to see more issues from.
+
+!["X releases on Y sources" button"](/images/inspector/inspector-issues-release-breakdown-modal.png)
```

Comment:
It might be nice to annotate this image, like pointing out that 2.3.0 on iOS has quite a low volume compared to its predecessor.

Reply:
Added in 112baf24db841b0a5f27f1a62b829d2921e3f52f